build/release/containers.sh

```diff
@@ -52,16 +52,60 @@ build() {
 
     [[ ${os} == "linux" ]] || return 0
 
-    tmpdir=$(mktemp -d)
-    trap "rm -fr $tmpdir" EXIT
-    cat <<EOF > $tmpdir/Dockerfile
+    echo "Building the rookd container"
+    tmpdir1=$(mktemp -d)
+    trap "rm -fr $tmpdir1" EXIT
+    cat <<EOF > $tmpdir1/Dockerfile
 FROM alpine:3.5
 RUN apk add --no-cache gptfdisk util-linux coreutils e2fsprogs
 COPY root /
 ENTRYPOINT ["/usr/bin/rookd"]
 EOF
-    build_artifact $os $arch $tmpdir rook/rookd rookd
-    rm -fr $tmpdir
+    build_artifact $os $arch $tmpdir1 rook/rookd rookd
+    rm -fr $tmpdir1
+
+    echo "Building the rook-client container"
+    tmpdir2=$(mktemp -d)
+    trap "rm -fr $tmpdir2" EXIT
+    cat <<EOF > $tmpdir2/Dockerfile
+FROM ubuntu
+RUN apt-get update && \
+    apt-get install -yq --no-install-recommends wget s3cmd kmod module-init-tools sudo
+COPY root /
+ENTRYPOINT ["/usr/bin/rook"]
+EOF
+    build_artifact $os $arch $tmpdir2 rook/rook-client rook
+    rm -fr $tmpdir2
+
+    echo "Building the toolbox container"
+    tmpdir3=$(mktemp -d)
+    cp toolbox/entrypoint.sh $tmpdir3
+    trap "rm -fr $tmpdir3" EXIT
+    cat <<EOF > $tmpdir3/Dockerfile
+FROM ubuntu:xenial
+RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 08b73419ac32b4e966c1a330e84ac2c0460f3994 \
+    && echo "deb http://download.ceph.com/debian-kraken/ xenial main" > /etc/apt/sources.list.d/ceph-kraken.list \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends \
```

Comment:
we will definitely get the benefit locally during development. On jenkins i'm not sure if the cleanup will remove all layers